app/src/main/res/layout/list_comments_item.xml

```diff
@@ -123,4 +123,34 @@
         android:textSize="@dimen/video_item_search_upload_date_text_size"
         tools:text="1 year ago" />
 
+    <TextView
+        android:id="@+id/showReplies"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/itemCommentContentView"
+        android:layout_alignEnd="@+id/itemCommentContentView"
+        android:layout_alignBottom="@+id/itemPublishedTime"
+        android:layout_marginStart="21dp"
+        android:layout_marginTop="0sp"
+        android:layout_marginEnd="0sp"
+        android:layout_marginBottom="0sp"
+        android:layout_toEndOf="@+id/itemPublishedTime"
+        android:text="@string/show_replies"
+        android:textAppearance="@style/TextAppearance.AppCompat.Small"
+        android:textSize="@dimen/video_item_search_upload_date_text_size" />
+
+    <androidx.recyclerview.widget.RecyclerView
```

Comment:
If I understood correctly https://github.com/TeamNewPipe/NewPipeExtractor/pull/703, a reply-comment could have replies itself, but this PR introduces support only for 1 level of reply depth. This would work fine for YouTube, but it would be problematic for other services, so the `replyRecyclerView` should be added dynamically. Doing this might not be best practice, since the layouts recycled in recycler views should not be built dynamically, but since the amount of comments we expect users to open is much smaller than the total number of comments, I don't think creating a few more views every time an expanded comment is shown would hurt performance whatsoever. Note: if you do this remember to cleanup the recycler view previously created, if there is one, before creating a new one.

Reply:
I am more inclined to use the first option as I am trying to keep the amount of new code minimal for future maintenance reasons. I was hoping that by reusing the 'InfoListAdapter` the comment views which were created would automatically have the reply handling code inside of it. How are the comment views which are being created here different than the ones which are created in the main comment holding view?